.github/workflows/test-integration.yml

```diff
@@ -45,6 +45,10 @@ jobs:
           node-version: 16
           cache: pnpm
           cache-dependency-path: "**/pnpm-lock.yaml"
+      - name: Install node-gyp globally
+        if: runner.os == 'Windows'
+        run: |
+          pnpm add -g node-gyp
```

Comment:
Is this required for this fix?

Reply:
Since pnpm is setup with the action, I believe the global install will be scoped to the action anyway so it seems fine to me 👍

Reply:
@valpinkman  I don't think we call the action on this workflow.
I can rewrite it quickly to add this instead : 
```
      - uses: ./tools/actions/composites/setup-test-desktop
        id: setup-test-desktop
 ```
 instead of : 
 ```
       - uses: pnpm/action-setup@v2
        with:
          version: latest
      - uses: actions/setup-node@v3
        with:
          node-version: 16
          cache: pnpm
          cache-dependency-path: "**/pnpm-lock.yaml"
    ```

or is it unneccesary ?

Reply:
`setup-test-desktop` does this

```
steps:
    - uses: pnpm/action-setup@v2
      with:
        version: latest
        dest: ${{ runner.temp }}/setup-pnpm
    - uses: actions/setup-node@v3
      with:
        node-version: 16
        cache: pnpm
        cache-dependency-path: "**/pnpm-lock.yaml"
    - uses: actions/setup-python@v4
      if: ${{ !inputs.skip_python }}
      with:
        python-version: "3.x"
    - uses: ruby/setup-ruby@v1
      if: ${{ !inputs.skip_ruby }}
      with:
        ruby-version: 2.7
        bundler-cache: true
```

I'm not sure you really need it actually. what you have is fine ;)

@henrily-ledger